src/components/common/MainMenu.js

```diff
@@ -67,6 +68,24 @@ export class MainMenu extends Component {
     return null;
   }
 
+  renderOfflineMenuItem = children => {
+    const { t } = this.props;
+
+    return (
+      <>
+        <Offline>
+          <Tooltip
+            placement="right"
+            title={t('You need an internet connection')}
+          >
+            <div>{React.cloneElement(children, { disabled: true })}</div>
```

Comment:
Does this work with more than one element? If not, then we should force it to always have one element somehow.